aws/table_aws_batch_job.go

```diff
@@ -0,0 +1,312 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/batch"
+	// "github.com/aws/aws-sdk-go-v2/service/batch/types"
+
+	batchv1 "github.com/aws/aws-sdk-go/service/batch"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBatchJob(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_batch_job",
+		Description: "AWS Batch Job",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValueException"}),
+			},
+			Hydrate: getBatchJob,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsBatchJobs,
```

Comment:
Could you please add the optional key quals for minimizing the result and the API time consumption?

We can filter the results by providing [these values](https://docs.aws.amazon.com/batch/latest/APIReference/API_ListJobs.html#API_ListJobs_RequestSyntax) as an input parameter of API call